pkg/proxy/nftables/helpers_test.go

```diff
@@ -207,6 +207,26 @@ func (tracer *nftablesTracer) addressMatches(ipStr, not, ruleAddress string) boo
 	}
 }
 
+func (tracer *nftablesTracer) addressesMatches(ipStr, not, ruleAddress string) bool {
+	ruleAddress = strings.ReplaceAll(ruleAddress, " ", "")
+	addresses := strings.Split(ruleAddress, ",")
+	if not == "!=" {
```

Comment:
You don't need to implement both "==" and "!=" matches; I only did that for `destAddrRegexp`/`sourceAddrRegexp` because we actually need it (for `ip daddr != 127.0.0.0/8` in the nodeport matching rules). But you don't ever generate any `!= { ... }` rules, so you can just simplify this by not bothering to implement that in the tracer.

Reply:
removed `==` check.